### PR TITLE
simulcast intro

### PR DESCRIPTION
**Pull request by arjunattam**

### CODE REVIEW

**Comment by arjunattam** [comment]
todo: make it show up in all platform foundation sidebar

File: common/adaptive-bitrate.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+Adaptive bitrate (ABR) refers to features that enable dynamic adjustments to video quality—to optimise for end-user experience under diverse network conditions. ABR ensures that every participant is able to consume the highest possible quality video in conferencing or streaming use-cases, based on their bandwidth constraints.
+
+In addition to network, ABR can also optimise for the right video quality based on the size of the video element. For example, a video call running on active speaker layout has larger video tiles that require higher quality video track. These adjustments can be made dynamically with adaptive bitrate.
```
Comment (gzerad) [inline common/adaptive-bitrate.md:3]: > or example, a video call running on active speaker layout has larger video tiles that require higher quality video track 
Maybe some diagrams would help here. Is it clear to people what active speaker layout is?

File: common/adaptive-bitrate.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+| 180p                  | 180p                      |
+| 240p                  | 240p                      |
+| 360p                  | 180p, 360p                |
+| 480p                  | 240p, 480p                |
+| 720p                  | 180p, 360p, 720p          |
+| 1080p                 | 270p, 540p, 1080p         |
+
+#### Enable via dashboard
+
+_This feature is in limited availability._
```
Comment (gzerad) [inline common/adaptive-bitrate.md:31]: what does this mean?

File: common/adaptive-bitrate.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+* JavaScript SDK
+* [Android SDK](/docs/android/v2/advanced-features/simulcast)
+* iOS SDK
+
+#### Automatic layer selection
+
+- Based on video tile size
+    - The `useVideo` hook in the 100ms React SDK automatically ensures that appropriate video layer is subscribed to, based on the size of the video element. For example, if the video element is 360 px in width, 360p or the closest layer will be selected.
+- Based on network quality: ABR will work alongside subscribe degradation and auto-downgrade video quality for peers
```
Comment (gzerad) [inline common/adaptive-bitrate.md:94]: should we say this is coming soon?

File: common/adaptive-bitrate.md (hunk 2 above)
Comment (arjunattam) [reply common/adaptive-bitrate.md:31]: the dashboard toggle is only available for a limited set of accounts. will be generally available in a week. should i frame this differently?

File: common/adaptive-bitrate.md (hunk 1 above)
Comment (arjunattam) [reply common/adaptive-bitrate.md:3]: i've added a demo video for this

File: common/adaptive-bitrate.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
+Adaptive bitrate (ABR) refers to features that enable dynamic adjustments to video quality—to optimise for end-user experience under diverse network conditions. ABR ensures that every participant is able to consume the highest possible quality video in conferencing or streaming use-cases, based on their bandwidth constraints.
```
Comment (sagargaonkar) [inline common/adaptive-bitrate.md:1]: Make this "Adaptive BitRate" to indicate what ABR stands for?

File: common/adaptive-bitrate.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+Simulcast configuration is opt-in and can be enabled on the role's template. The role's publish video quality determines video quality layers on simulcast. For example, a role configured to publish at 720p can simulcast 180p, 360p and 720p layers.
+
+| Video publish quality | Possible simulcast layers |
+|-----------------------|---------------------------|
+| 180p                  | 180p                      |
+| 240p                  | 240p                      |
+| 360p                  | 180p, 360p                |
+| 480p                  | 240p, 480p                |
+| 720p                  | 180p, 360p, 720p          |
+| 1080p                 | 270p, 540p, 1080p         |
```
Comment (sagargaonkar) [inline common/adaptive-bitrate.md:27]: Put these in reverse order?

File: common/adaptive-bitrate.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+<br/>
+
+#### Manual layer selection
+
+The 100ms client-side SDKs provide methods to set a preferred quality layer for a remote peer's video track. See docs for your preferred platform:
+
+* [JavaScript](/docs/javascript/v2/advanced-features/simulcast)
+* [iOS](/docs/ios/v2/advanced-features/simulcast)
+* [Android](/docs/android/v2/advanced-features/simulcast)
```
Comment (saikatmitra91) [inline common/adaptive-bitrate.md:90]: no docs for API 🤔 ?

File: common/adaptive-bitrate.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+
+## ABR in conferencing
+
+Peers in 100ms rooms can publish multiple video quality levels simultaneously. This is called “simulcast” in 100ms. Peers that consume these video tracks can upgrade or downgrade video quality.
+
+You can enable simulcast on the publishing role's template, and use manual or automatic layer changes on the subscriber's side.
+
+### Publisher-side configuration
+
+Simulcast configuration is opt-in and can be enabled on the role's template. The role's publish video quality determines video quality layers on simulcast. For example, a role configured to publish at 720p can simulcast 180p, 360p and 720p layers.
```
Comment (saikatmitra91) [inline common/adaptive-bitrate.md:18]: role's template 

template is something else in our world.  Role is associated to a template till now. role's template might be confusing. wdyt ?

**Review by saikatmitra91 — CHANGES_REQUESTED** [review]

File: common/adaptive-bitrate.md (hunk 6 above)
Comment (arjunattam) [reply common/adaptive-bitrate.md:90]: API docs are not relevant for the subscriber side. i have linked them on the publisher side for template update

File: common/adaptive-bitrate.md (hunk 7 above)
Comment (arjunattam) [reply common/adaptive-bitrate.md:18]: yup, fixing

**Comment by arjunattam** [comment]
thanks for the workaround on the code block formatting, @KaustubhKumar05. merging this PR

**Pull request merged**